### PR TITLE
update apifox

### PR DESCRIPTION
**Pull request by chriscp-cat**
Closes #XXXX

Relates to #XXXX

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/apifox-portable.json [hunk 1]
```diff
@@ -24,12 +24,12 @@
     ],
     "checkver": {
         "url": "http://cdn.apifox.cn/download/latest.yml",
-        "regex": "^version:[\\s]+([\\d.]+)"
+        "regex": "version:[\\s]+([\\d.]+)[\\s\\S]*?url:[\\s]+http://(?<url>.*?\\.exe)"
```
Comment (JaimeZeng) [inline bucket/apifox-portable.json:27]:
```suggestion
        "regex": "(?sm)version:[\\s]+([\\d.]+).*?http://(?<url>.*?\\.exe)"
```

File: bucket/apifox-portable.json [hunk 2]
```diff
@@ -24,12 +24,12 @@
...
     },
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "http://apifox-generic.pkg.coding.net/apifox/apifox-desktop/Apifox-$version.exe#/dl.7z",
+                "url": "http://$matchUrl",
```
Comment (JaimeZeng) [inline bucket/apifox-portable.json:32]:
```suggestion
                "url": "http://$matchUrl#/dl.7z",
```

File: bucket/apifox.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "depends": "sudo",
     "architecture": {
         "64bit": {
-            "url": "http://apifox-generic.pkg.coding.net/apifox/apifox-desktop/Apifox-2.0.4.exe#/installer.exe",
+            "url": "http://cdn.apifox.cn/download/2.0.4/Apifox-2.0.4.exe",
```
Comment (JaimeZeng) [inline bucket/apifox.json:12]:
```suggestion
            "url": "http://cdn.apifox.cn/download/2.0.4/Apifox-2.0.4.exe#/installer.exe",
```

File: bucket/apifox.json [hunk 4]
```diff
@@ -29,12 +29,12 @@
     ],
     "checkver": {
         "url": "http://cdn.apifox.cn/download/latest.yml",
-        "regex": "^version:[\\s]+([\\d.]+)"
+        "regex": "version:[\\s]+([\\d.]+)[\\s\\S]*?url:[\\s]+http://(?<url>.*?\\.exe)"
```
Comment (JaimeZeng) [inline bucket/apifox.json:32]:
```suggestion
        "regex": "(?sm)version:[\\s]+([\\d.]+).*?http://(?<url>.*?\\.exe)"
```

File: bucket/apifox.json [hunk 5]
```diff
@@ -29,12 +29,12 @@
...
     },
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "http://apifox-generic.pkg.coding.net/apifox/apifox-desktop/Apifox-$version.exe#/installer.exe",
+                "url": "http://$matchUrl",
```
Comment (JaimeZeng) [inline bucket/apifox.json:37]:
```suggestion
                "url": "http://$matchUrl#/installer.exe",
```

**Comment by chriscp-cat** [comment]
才知道有(?sm)这种东西

**Comment by JaimeZeng** [comment]
All changes look good.

Wait for review from human collaborators.
### apifox-portable

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

### apifox

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Pull request merged**